2-mandatory/6-budgets.js

```diff
@@ -11,8 +11,12 @@ Should give return the answer of 62600.
 **/
 
 function getBudgets(peopleArray) {
+  return peopleArray.reduce((sum, person) => {
+    return (sum += person.budget);
```

Comment:
If you remove the internal {} you shouldn't need the second return statement

Reply:
I typically write out the full method beforehand to avoid making mistakes and then I leave like that.  I will try to use the shorthand version more from now on.